<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>urunguncelleme</class>
 <widget class="QDialog" name="urunguncelleme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bisküvi Ürün Güncelleme</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ürün Ara:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uAraTxt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="guncelleBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Güncelle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="silBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ürün Türü:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ürün Adi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birim Fiyati:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UTtxt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="UAdtxt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bFiyatdsb">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
